src/main/java/org/jenkinsci/plugins/workinghours/model/ExcludedDate.java

```diff
@@ -305,7 +304,7 @@ public void setHolidayId(String holidayId) {
 
         public Date(JSONObject jsonObject, boolean isEnd) {
             this.date = jsonObject.getString(FIELD_DATE);
-            if(isEnd){
+            if (isEnd) {
```

Comment:
Super nit, but I noticed the parameter is called `isEnd` here, but `isEndDate` in the function below.  Does it make sense to call them both isEndDate?